docs/serving/samples/hello-world/helloworld-java-spring/README.md

```diff
@@ -200,21 +200,17 @@ folder) you're ready to build and deploy the sample app.
 
     ```shell
     kubectl get ksvc helloworld-java-spring \
-       --output=custom-columns=NAME:.metadata.name,DOMAIN:.status.domain
+       --output=custom-columns=NAME:.metadata.name,URL:.status.url
```

Comment:
I think this should also be left alone -- I think that's one of the changes with the move from the alpha API to the beta API, and since we don't want to make that update here, I believe this should be backed out too.